setup.cfg

```diff
@@ -39,3 +41,21 @@ exclude =
 # module-attr: pytype cannot understand the namespace packages by Google.
 # attribute-error: Data classes in grpc module doesn't specify attributes.
 disable = "import-error,module-attr,attribute-error"
+
+[mypy]
+cache_dir = /dev/null
+disable_error_code = 
+    import,
```

Comment:
Please leave a comment explaining why these two errors need to be silenced.

Reply:
Done